### DISCUSSION TITLE
The image remains static and does not change upon selecting different variants.

### DISCUSSION THREAD

**Discussion by flonUK** [swellstores/community#481] [Help] [answered]
I am encountering an issue on my product page that I hope you can assist me with. The page features multiple product variants, each linked to a specific image. Ideally, when a customer selects a variant, the corresponding image should automatically update to reflect their choice. However, this functionality is not working as expected.

Link: https://forloveofnature.co.uk/products/the-single-2-0
Admin dashboard screenshot: 
![image](https://github.com/swellstores/community/assets/156616226/2d8ad8b9-c3c9-4843-b068-e21a4d60b29d)

Wonder if anyone has fixed this or have a workaround please?

Thanks

**Comment by logeshswell** [comment, marked as answer]
> Ideally, when a customer selects a variant, the corresponding image should automatically update to reflect their choice.

I think it's not programmed in such a way for horizon theme. But, you can retrieve corresponding image URLs from `variants.image` value and render it using button click listener.
